src/ops/ScriptOps.ts

```diff
@@ -198,20 +201,24 @@ export default (state: State): Script => {
     ): Promise<ScriptExportInterface> {
       return exportScriptByName({ scriptName, state });
     },
-    async exportScripts(): Promise<ScriptExportInterface> {
-      return exportScripts({ state });
+    async exportScripts(
```

Comment:
Ignore this comment. It is a test review comment to see if force push delete its or keeps it part of the history.